tests/foreman/ui/test_computeresource_gce.py

```diff
@@ -0,0 +1,158 @@
+"""Test for Compute Resource UI
+
+:Requirement: Computeresource GCE
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from fauxfactory import gen_string
+from nailgun import entities
+from pytest import skip
+from robottelo import ssh
+from robottelo.config import settings
+from robottelo.constants import (
+    COMPUTE_PROFILE_SMALL,
+    FOREMAN_PROVIDERS,
+    GCE_EXTERNAL_IP_DEFAULT,
+    GCE_MACHINE_TYPE_DEFAULT,
+    GCE_NETWORK_DEFAULT
+)
+from robottelo.decorators import bz_bug_is_open, fixture, setting_is_set, tier2
+
+
+if not setting_is_set('gce'):
+    skip('skipping tests due to missing gce settings', allow_module_level=True)
+
+
+class GCECertNotFoundError(Exception):
+    """An exception to raise when GCE Cert json is not available for creating GCE CR"""
```

Comment:
Is there a reason to create and raise custom exception here?

I see similar pattern is employed in `tests/foreman/cli/test_satellitesync.py`, but usually we raise one of existing exceptions, or simply `Exception`.

Reply:
No special reason, but we get to know the issue by looking at the exception name itself.

This kind of exceptions are being employed from ages in satellite6 QE and I am very used to it :)